### DISCUSSION TITLE
Search Hit Count

### DISCUSSION THREAD

**Discussion by hpcnetworks** [sepinf-inc/IPED#1963] [General]
Hi @lfcnassif !

On discussion #1226 you said that were about to try to compute the exact number of hits for the queries. Did it move forward?
I need to answer the number os hits (ot the number of text fragments with hits), but I only have this info per result item. On a specific case, I have 807 items... :(((

![image](https://github.com/sepinf-inc/IPED/assets/121892411/9c469e3d-d343-480b-8a6f-659200be40c6)

Thanks in advance!

Bruno

**Comment by lfcnassif** [comment]
Unfortunately not, other things took higher priority over it and are still taking...

**Reply by hpcnetworks** [reply]
Thanks, for the quick reply! I will figure out how to deal with it... rsrs
Best Regards,
Bruno
